src/dataEntryApp/views/subjectDashBoard/components/subjectDashboardProgramView.js

```diff
@@ -0,0 +1,266 @@
+import React, { Fragment } from "react";
+import Grid from "@material-ui/core/Grid";
+import Fab from "@material-ui/core/Fab";
+import { makeStyles } from "@material-ui/core/styles";
+import { Paper } from "@material-ui/core";
+import ExpansionPanel from "@material-ui/core/ExpansionPanel";
+import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
+import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import Typography from "@material-ui/core/Typography";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import moment from "moment/moment";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableRow from "@material-ui/core/TableRow";
+import ErrorIcon from "@material-ui/icons/Error";
+
+import Button from "@material-ui/core/Button";
+
+const useStyles = makeStyles(theme => ({
+  enrollButtonStyle: {
+    marginBottom: theme.spacing(2),
+    marginRight: theme.spacing(2)
+  },
+  root: {
+    flexGrow: 1,
+    padding: theme.spacing(2)
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: "left"
+  },
+  programStatusStyle: {
+    color: "red",
+    backgroundColor: "#FFB6C1",
+    borderRadius: "5px"
+  },
+  expansionHeading: {
+    fontSize: theme.typography.pxToRem(15),
+    flexBasis: "33.33%",
+    flexShrink: 0
+  },
+  listItem: {
+    paddingBottom: "0px",
+    paddingTop: "0px"
+  },
+  table: {
+    border: "1px solid rgba(224, 224, 224, 1)"
+  },
+  abnormalColor: {
+    color: "#ff4f33"
+  }
+}));
+
+const ProgramView = ({ programData }) => {
+  const classes = useStyles();
+  const [expanded, setExpanded] = React.useState(false);
```

Comment:
You are using expanded state to hold two type of variables: string and boolean. This is confusing.